src/lava/magma/core/callback_fx.py

```diff
@@ -53,3 +54,23 @@ def post_run_callback(self,
                           board: NxBoard = None,
                           var_id_to_var_model_map: dict = None):
         pass
+
+
+class IterableCallBack(NxSdkCallbackFx):
+    """NxSDK callback function to execute iterable of function pointers
+    as pre and post run."""
+
+    def __init__(self,
+                 pre_run_fxs: Iterable = [],
+                 post_run_fxs: Iterable = []) -> None:
```

Comment:
To fix the codacy issue we need to take out `pre_run_fxs: Iterable = []` and instead put in `pre_run_fxs: Iterable` and then 
```
if not pre_run_fxs:
    pre_run_fxs = []
```

same for `post_run_fxs`.